src/Components/Home.js

```diff
@@ -1,28 +1,37 @@
 import React from 'react';
 import { connect } from 'react-redux';
-import { NavLink } from 'react-router-dom';
+import { logout } from '../redux/actions/actionCreators';
+import Sidebox from './Sidebox';
+import World from './World';
```

Comment:
I don't think we need World, either. I'll be replacing it with what I've written. :))